portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Comment {
+
+    // Name of user leaving the comment
+    private String name;
+
+    // Location of user
+    private String location;
+
+    // Text content of comment
+    private String content;
+
+    public Comment(String name, String location, String content) {
```

Comment:
Here it would be good to also update the names and to give a short description of what each of the arguments is supposed to represent. Say, something like

```
/* Comment:
  user_name: The name (full name? user ID?) of the user posting the comment.
  user_location: Location of the comment poster.  Country? State? What do you expect?
  content: the comment.
*/
```